Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -707,7 +708,7 @@
     staticRecipes:
       - BoxLethalshot
       - BoxShotgunFlare
-      - BoxShotgunPractice 
```

Comment:
Also pointyhat for changing random whitespace and making me wonder why a boots PR is changing shotgun shells